test/e2e/coverage_docs/spread_by_region_test.md

```diff
@@ -0,0 +1,6 @@
+### spread by region e2e test coverage analysis
+
+| Test Case                                                     | E2E Describe Text                                         | Comments                                                                                                                 |
+|---------------------------------------------------------------|-----------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| Verifying the deployment spread across multiple regions       | check whether deployment is scheduled to multiple regions | [Multi region HA support](https://karmada.io/zh/docs/userguide/scheduling/resource-propagating/#multi-region-ha-support) |
+| Updating the propagation policy and verifying its constraints | update propagation policy to propagate to one region      |                                                                                                                          |
```

Comment:
We need to map the test cases to `ginkgo.It`. If we find that `ginkgo.It` is not appropriate, we can directly modify it.